vmm/src/lib.rs

```diff
@@ -173,6 +180,9 @@ pub fn start_vmm_thread(
     let thread = thread::Builder::new()
         .name("vmm".to_string())
         .spawn(move || {
+            // Apply seccomp filter for VMM thread.
```

Comment:
What about the other threads? That start here, API thread?

Reply:
Well I wanted to apply it to the most important thread first, but yes we need other filtering for the API thread.